ci-operator/jobs/openshift/builder/openshift-builder-presubmits.yaml

```diff
@@ -63,3 +63,204 @@ presubmits:
+          valueFrom:
+            configMapKeyRef:
+              key: master.yaml
+              name: ci-operator-openshift-origin
+        - name: OVERRIDE_SPEC
+          value: '{"test_base_image": {"cluster": "", "namespace": "ci", "name": "src-cache-origin",
+            "tag": "master"}}'
+        image: ci-operator:latest
+        name: test
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-gcp-crio
+          name: job-definition
+          subPath: cluster-launch-e2e.yaml
+        - mountPath: /usr/local/e2e-gcp-crio-cluster-profile
+          name: cluster-profile
+      serviceAccountName: ci-operator
+      volumes:
+      - configMap:
+          name: prow-job-cluster-launch-e2e
+        name: job-definition
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-gcp
+          - configMap:
+              name: cluster-profile-gcp-crio
+    trigger: ((?m)^/test( gcp-crio),?(\s+|$))
+  - agent: jenkins
+    always_run: true
+    branches:
+    - master
+    context: ci/openshift-jenkins/extended_conformance_install
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_origin_extended_conformance_install
+    rerun_command: /test extended_conformance_install
+    trigger: ((?m)^/test( all| extended_conformance_install),?(\s+|$))
+  - agent: jenkins
+    always_run: false
+    branches:
+    - master
+    context: ci/openshift-jenkins/extended_builds
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_origin_extended_builds
+    rerun_command: /test extended_builds
+    run_if_changed: ^(pkg/oc/cli/startbuild/startbuild.go)|^(pkg/build)|^(pkg/cmd/infra/builder)|^(test/extended/builds)|^(test/extended/testdata/builds)|^(vendor/github.com/openshift/source-to-image)|^(vendor/github.com/openshift/imagebuilder)
+    trigger: ((?m)^/test extended_builds,?(\s+|$))
+  - agent: jenkins
+    always_run: true
+    branches:
+    - master
+    context: ci/openshift-jenkins/extended_clusterup
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_origin_extended_clusterup
+    rerun_command: /test extended_clusterup
+    trigger: ((?m)^/test extended_clusterup,?(\s+|$))
+  - agent: jenkins
+    always_run: false
+    branches:
+    - master
+    context: ci/openshift-jenkins/extended_image_ecosystem
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_origin_extended_image_ecosystem
+    rerun_command: /test extended_image_ecosystem
+    run_if_changed: ^(test/extended/image_ecosystem)|^(examples/db-templates)|^(examples/quickstarts)|^(examples/image-streams)|^(examples/jenkins)|^(vendor/github.com/openshift/source-to-image)|^(vendor/github.com/openshift/imagebuilder)
+    trigger: ((?m)^/test extended_image_ecosystem,?(\s+|$))
+  - agent: kubernetes
```

Comment:
why are you building cross?

Reply:
in theory, to cover ppc64, does cross handle that?